### DISCUSSION TITLE
Slide bar in frame?

### DISCUSSION THREAD

**Discussion by cpadlab** [TomSchimansky/CustomTkinter#890] [Q&A]
Is there a way that all the content (be it labels, buttons, ...) of a frame that sticks out or is not visible due to lack of space can be seen with a slider bar or something?

**Comment by JardarIversen** [comment]
In #904 i asked if scrollbars will be added soon, and I think thats what you are looking for here. Best to wait until that is implemented imo

**Reply by zSynctic** [reply]
yeah it already released its called ```CTkScrollableFrame```

**Comment by avalon60** [comment]
> Is there a way that all the content (be it labels, buttons, ...) of a frame that sticks out or is not visible due to lack of space can be seen with a slider bar or something?

See widget, bottom left of image:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/353eab27-dc66-433f-8fe6-5da95e25fd8f)
